src/pack.rs

```diff
@@ -40,7 +40,14 @@ impl FHESerialize for PublicKey {
     where
         Self: Sized,
     {
-        deserialize(bytes).map_err(|_| FheError::InvalidEncoding)
+        let result = deserialize(bytes).map_err(|_| FheError::InvalidEncoding);
+
+        if let Err(ref e) = result {
+            // This is a hack to support the old serialization format.
+            println!("Failed to deserialize the public key");
+        }
```

Comment:
What's going on here? We print the deserialization error, but it doesn't look like any kind of hack or support for older formats.

Reply:
Oh crap sorry this should not have been committed. I was debugging because solidity doesn't show anything.